fastlane/Fastfile

```diff
@@ -146,6 +136,12 @@ platform :android do
     end
 
     lane :google_play_distribute do |options|
+        version_number_string = options[:version_number_string]
+        setup_ci(force: true)
+        update_version_number version_number_string:version_number_string
+        tag = "#{version_number_string}-android"
+        add_git_tag(tag: tag, force: true)
+        push_git_tags(tag: tag, force: true)
```

Comment:
Would it not be worth doing the tag as part of `update_version_number`

Reply:
This way it tags the version commit rather than the previous commit. Can be done in that lane but works here just as well. As mentioned before though this is all just v1 to get something basic working and needs revisiting